arviz/rcparams.py

```diff
@@ -96,6 +96,7 @@ def _validate_probability(value):
     "plot.bokeh.figure.dpi": (60, _validate_positive_int),
     "plot.bokeh.figure.width": (500, _validate_positive_int),
     "plot.bokeh.figure.height": (500, _validate_positive_int),
+    "plot.bokeh.show": (True, _make_validate_choice({True, False}, typeof=bool)),
```

Comment:
I think it would be best to create a new validation function for booleans. The validate choice relies on calling `cls(value)` which in the case of booleans would mean accepting nearly everything mapping everyhting to `True`. In fact, the string `"false"` is mapped to `True` doing this. 

I think that only booleans `True`, `False` and strings `true` and `false` (ignoring cases) should be accepted. The string will be what is generally used (i.e. when reading an arvizrc file), and the booleans will be used when using `rc_context`.

Reply:
Ok I'll change this to a string!

Reply:
using `make_validate_choice` either with boolean or with string won't work, as you say at some point is has to turn into boolean. I am proposing to create a new `_validate_boolean` that returns `True` (as boolean) if value is `True` or `"true"`, returns `False` (as boolean) if value is `False` or `"false"` and raises an error otherwise.